articles/container-apps/how-to-integrate-with-azure-front-door.md

```diff
@@ -216,7 +216,7 @@ az network private-endpoint-connection approve --id <PRIVATE_ENDPOINT_CONNECTION
 
 ## Add a route
 
-Run the following command to map the endpoint you created earlier to the origin group.
+Run the following command to map the endpoint you created earlier to the origin group. Private endpoints on Azure Container Apps only support inbound HTTP traffic. TCP traffic is not supported.
```

Comment:
Expand the hidden content below to see the context: in `az afd route create`, we specify `--supported-protocols Http Https`.